build/workbox.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+var fse = require('fs-extra')
+var config = require('./workbox.config.json')
+var swBuild = require('workbox-build')
+
+var mdToHtml = (manifestEntries) => manifestEntries.map((entry) => {
+  if (entry.url.startsWith('docs/') && entry.url.indexOf('.md') > -1) {
+    var suffix = entry.url.indexOf('index') > -1 ? '.html' : '/index.html'
+    entry.url = entry.url.replace(/.md/g, '') + suffix
+  }
+  return entry
+})
+
+var workboxSWSrcPath = require.resolve('workbox-sw')
+var workboxSWDestPath = 'assets/js/vendor/workbox-sw.js'
+fse.exists(workboxSWDestPath, function (e) {
+  if (!e) {
+    fse.copy(workboxSWSrcPath, workboxSWDestPath)
+    console.log('copied workbox service worker.')
+  }
+})
+
+config.manifestTransforms = [mdToHtml]
```

Comment:
Why do we need workbox to do MD -> HTML?

The _gh_pages folder already has the html files.